### ISSUE TITLE
WIN11 make build error: it  said there is no dictionary target/release/libtokenizers.a

### ISSUE THREAD

**Issue by GinWithoutA** [daulet/tokenizers#26]
I performed the "make build" operation on Windows 11. The first step was completed without any errors. However, in the second step, when executing "cp target/release/libtokenizers.a", an error occurred as the file could not be found. I would like to ask how this problem can be solved. I have installed MinGW64 and the Rust environment, so there should be no problem with the environment.
<img width="411" alt="微信图片_20240829184203" src="https://github.com/user-attachments/assets/15c58974-74ac-4cbf-a202-851caa90dc09">

**Comment by daulet** [comment]
not familiar with windows builds, check where rust puts its built artifacts on windows, perhaps it is in a different directory from `target`

**Comment by GinWithoutA** [comment]
Thank you so much!. I searched using Everything but couldn't find the `libtokenizers.a` file and I am unsure how to resolve this issue. Later, I switched to an Ubuntu platform to try again, but now I encounter a new problem. When executing `go build .` from the `Makefile`, it reports "No Such File Or Directory" error. Running `go build -ldflags` directly also results in the same error. I would like to ask how to resolve this issue as I have no clue.
<img width="635" alt="微信图片_20240830161051" src="https://github.com/user-attachments/assets/cfa4012f-c11b-4d56-ad5c-4f9f3f4407c7">
